packages/office-ui-fabric-react/src/components/Dropdown/Dropdown.test.tsx

```diff
@@ -118,6 +118,19 @@ describe('Dropdown', () => {
       expect(titleElement.textContent).toEqual('2');
     });
 
+    it('does clear when the selectedKey is null', () => {
```

Comment:
Shouldn't combo box have the same test given that it also takes in selectedKey?

Reply:
ok done